src/source-updater.js

```diff
@@ -78,7 +83,15 @@ export default class SourceUpdater {
   appendBuffer(bytes, done) {
     this.processedAppend_ = true;
     this.queueCallback_(() => {
-      this.sourceBuffer_.appendBuffer(bytes);
+      try {
+        this.sourceBuffer_.appendBuffer(bytes);
+      } catch (error) {
+        this.sourceBuffer_.trigger({
```

Comment:
I guess we should double check that the error type is correct here right? Just in case it isn't actually a QuotaExceededError. Alternatively, I'm not sure we need to catch the error here since it's already handled in media-sources

Reply:
This must exist to handle errors thrown by fmp4 streams (where media-sources is not used).

Added the error checking.